src/Control/Director.php

```diff
@@ -76,40 +80,47 @@ class Director implements TemplateGlobalProvider
     private static $alternate_base_folder;
```

Comment:
Maybe I misunderstood when I commented - is this the local file system path rather than a portion of the URL? If that's the case then I agree it can be included